### DISCUSSION TITLE
Request for Feedback re: v0.3.1-Beta

### DISCUSSION THREAD

**Discussion by Thinkersbluff** [Thinkersbluff/DGUS-Reloaded_for_CR6-DWIN-SET_Component#7] [Q&A] [answered]
[Version 0.3.1-Beta of  DGUS-Reloaded, CR6Community Edition was released](https://github.com/Thinkersbluff/DGUS-reloadedForKlipper_CR6/releases/tag/v0.3.1) on 11 Jan 2023.

Please comment below, if you have any feedback on this version of the firmware.

I would particularly appreciate actionable feedback on these questions of mine:

1. Do you experience any of the "known issues" documented in the Release Notes?
    -  this info may help me to troubleshoot and resolve those issues

2. Are there specific details in the repository readme files or readme.txt files that you find particularly confusing or missing?
    - I am hoping to develop a more robust set of documentation to support this solution.  I will need your help to know which missing info would have been helpful and which existing info was confusing or even incorrect.

3. Does using the modified DGUS-Reloaded version of Klipper on your system create other issues on your system? (e.g. impacts on other add-ins, problems with Moonraker Update Manager, etc.)

4. Does this version support your preferred 3D-printing workflow, now?  If not, what new feature would you like to see prioritized at the next release, and why?
  - I am still thinking about how best to engage with you - the CR6Community - to guide the further development of this UI.
  - I am largely paced and constrained by my limited knowledge of these key components of this system; e.g. python syntax and programming, the DWIN serial interface, Klipper macros, the Moonraker API.  I am also new to Git and GitHub, so may not be configuring or using the repo features well.

All feedback gratefully received!

Please add your comments below.

**Comment by cylonr** [comment]
I allready test the 0.3.1 Beta vertion and i didn´t finbd any issue.

One ideia: e-stepps option. To purge 100mm

@: sorry for my poor english

**Reply by Thinkersbluff** [reply]
So if you power up, go to Move and Home all, the system does not Shutdown, for you?

**Reply by cylonr** [reply]
No. It works perfect. If you want i can make a video.

**Reply by Thinkersbluff** [reply]
That would be great, thank you! 
I have a persistent problem on my printer. Since you do not, I must have a configuration issue or a macro conflict in my setup. 
I guess it's a good thing that your system is not affected. Now I wonder what is different between our two systems.

**Reply by henriquevianasilva** [reply]
I don't have this problem either.

Installed on an ERA Board.

**Reply by Thinkersbluff** [reply]
I did a little Klippy logging and Googling. Looks like my BTT SKR CR6 board may be struggling with TMC driver timing issues. 
Sad to say, I have pulled that board out and will now swap in my spare ERA board. 

Fingers crossed this fixes it for me. I will update this post with the outcome.

**Reply by henriquevianasilva** [reply]
Need edit printer.cfg
Do you have pancake? vref calibrate

**Reply by Thinkersbluff** [reply]
Vref may be a problem in my future. 
I do have a pancake motor on the extruder and have not yet tried to print for multiple hours with this ERA board. Do you have any suggestions for setting extruder Vref?

**Reply by henriquevianasilva** [reply]
adjusting the Vref was simple, just set the extruder to 0.5v

https://support.bondtech.se/Guide/00.%20Stepper%20Current%20Adjustment%20On%20CR-10S/46.html


I'll share my makeconfig settings later.

**Reply by henriquevianasilva** [reply]
![image](https://user-images.githubusercontent.com/67318712/212470380-6ae5730e-cea9-4caf-beeb-33e293cc88ce.jpeg)


Makeconfig for ERA BOARD v1.1.0.3

**Reply by Thinkersbluff** [reply]
Thanks for sharing. It took me a while to work my way through a lot of bad guesses first, before I tried that combo too. (eg, I was not sure whether the serial interface baud rate was a reference to the display serial interface or the mcu to Klipper serial interface (it is of course the latter), because of where the question appears in the modified menu. )

I might try to clarify that menu.

**Comment by Thinkersbluff** [comment, marked as answer]
OK, so it took me a while to guess the right combination of settings, but now I have DGUS-Reloaded_CR6CommunityEdition running on my CR6-SE with the ERA 1.1.0.3 board in it.  

- No more problems with Shutdown on the first X and Y home.
- Nozzle temperature (PT1000 thermistor) is no longer reading several degrees higher than the bed, at room temperature.
- Nozzle temperature is no longer bouncing up and down +/-3 degrees, the whole time I am printing...

I guess that is proof that the Creality ERA board is better hardened against EMI/EMC troubles.
Good news for folks who wish they could still buy the BTT board...

For reference, I have now added both makemenu config screen captures and the resulting Klipper.bin files to the Related Changes folder, for the convenience of new users.
